### ISSUE TITLE
Need admin rights to Collector repos

### ISSUE THREAD

**Issue by tigrannajaryan** [open-telemetry/community#272]
Can one of the org admins give me admin rights to https://github.com/open-telemetry/opentelemetry-collector and https://github.com/open-telemetry/opentelemetry-collector-contrib

As a maintainer I need to be able to set branch protection rules and other settings which are currently not available to me.

**Comment by tigrannajaryan** [comment]
@open-telemetry/technical-committee I am not sure who is the right person to handle this, so tagging TC for help.

**Comment by SergeyKanzhelev** [comment]
What changes do you need to make? Just checked, they look aligned with https://github.com/open-telemetry/community/blob/master/docs/how-to-configure-new-repository.md

**Comment by tigrannajaryan** [comment]
For example: Collector core repo uses TravisCI, Contrib repo uses CircleCI. I wanted to look into making this uniform and using only one (most likely CircleCI) but I cannot change these settings. If I remember correctly they should in branch protection settings somewhere.

Is there a reason maintainers should not be able to make these changes? It looks unnecessarily restrictive to me. IMHO maintainers should have full Admin rights over their SIG repos.

**Comment by SergeyKanzhelev** [comment]
let's separate two. Feedback on repo set up and anything you need right away to unblock you.

Here are protection settings:

Collector:
![image](https://user-images.githubusercontent.com/9950081/70662072-60623e00-1c1a-11ea-91fc-e3f560230120.png)


![image](https://user-images.githubusercontent.com/9950081/70662128-7e2fa300-1c1a-11ea-85b6-4f2c348fbce1.png)


Contrib: 

![image](https://user-images.githubusercontent.com/9950081/70662079-66581f00-1c1a-11ea-951d-6087ce862d4a.png)


![image](https://user-images.githubusercontent.com/9950081/70662150-87207480-1c1a-11ea-99bf-f64960a2e800.png)

I'm not sure if you you can register webhooks without being admin. Happy to make you one temporarily to set it up. When do you plan to work on it?


As for permissions question, would you mind creating a separate issue?

**Comment by tigrannajaryan** [comment]
>Feedback on repo set up and anything you need right away to unblock you

@SergeyKanzhelev for now please change https://github.com/open-telemetry/opentelemetry-collector-contrib to require 2 approvals before merging. I believe this is already the case for https://github.com/open-telemetry/opentelemetry-collector

For the rest of changes I need to experiment, so it will be very cumbersome if I don't have the rights directly. I can wait for now.

**Comment by SergeyKanzhelev** [comment]
To make it easier I made you an admin for now. We did it before when were kicking off Python.

**Comment by SergeyKanzhelev** [comment]
closing. Will remove admin permissions sometime in January

**Comment by tigrannajaryan** [comment]
Thanks.
